### PR TITLE
Change .format to f string on asv/machine.py

### PR DESCRIPTION
**Pull request by asnramos**
xref #1176 #1077 
Change f-strings instead .format() on asv/machine.py

### CODE REVIEW

File: asv/machine.py [hunk 1]
```diff
@@ -53,8 +53,7 @@ def load(cls, machine_name, _path=None):
                 return d[list(d.keys())[0]]
 
         raise util.UserError(
-            "No information stored about machine '{0}'. I know about {1}.".format(
-                machine_name, util.human_list(d.keys())))
+            f"No information stored about machine '{machine_name}'. I know about {util.human_list(d.keys()))}."
```
Comment (LucyJimenez) [inline asv/machine.py:56]:
```suggestion
            f"No information stored about machine '{machine_name}'. I know about {util.human_list(d.keys()))}.")
```

You miss an `)` to close `util.UserError( )`.

**Review by LucyJimenez — CHANGES_REQUESTED** [review]
Great job, @asnramos. I leave you just one comment :)

**Comment by datapythonista** [comment]
@asnramos CI is red, one line is longer than the maximum we allow, 99 characters. Can you check and split it in two lines please?

**Review by LucyJimenez — CHANGES_REQUESTED** [review]

**Review by LucyJimenez — APPROVED** [review]
LGTM! 🚀

**Comment by datapythonista** [comment]
Thanks @asnramos, really nice

**Pull request merged**